chart/kafka-connector/values.yaml

```diff
@@ -3,6 +3,7 @@ replicas: 1
 gateway_url: http://gateway.openfaas:8080
 topics: faas-request
 print_response: true
+print_response_body: true
```

Comment:
current behaviour doesn't print the response body, consider setting the default value to false

Reply:
I would agree with this

Reply:
Thanks for the review Keiran, added commit which references that 👍

Reply:
Will squash when everything looks fine